account_invoice_date_due/README.rst

```diff
@@ -0,0 +1,36 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
The manifest and some headers say the license is LGPL, but here in the README and some other headers say AGPL.